scanner/Makefile

```diff
@@ -60,6 +60,7 @@ GO_BUILD_FLAGS = CGO_ENABLED=$(CGO_ENABLED) GOOS=$(GOOS) GOARCH=$(GOARCH)
 GO_BUILD_CMD   = $(GO_BUILD_FLAGS) go build \
                    -trimpath \
                    -ldflags="-X github.com/stackrox/rox/scanner/internal/version.Version=$(TAG)" \
+                   -ldflags="-X github.com/stackrox/rox/scanner/internal/version.VulnerabilityVersion=$(SCANNER_VULNERABILITY_VERSION)" \
```

Comment:
does order not matter? sometimes it does in the Makefile (I think), so just confirming `SCANNER_VULNERABILITY_VERSION` will be evaluated in time for this even though it's declared below

Reply:
Due to the `=` in the assignment this will be evaluated at reference time.